packages/main/src/components/ObjectPage/ObjectPageAnchorTab.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable no-underscore-dangle */
+import ifDefined from '@ui5/webcomponents-base/dist/renderer/ifDefined.js';
+import { html, setSuffix, setTags } from '@ui5/webcomponents-base/dist/renderer/LitRenderer.js';
+import { ThemingParameters } from '@ui5/webcomponents-react-base';
+import Icon from '@ui5/webcomponents/dist/Icon.js';
+import Tab from '@ui5/webcomponents/dist/Tab.js';
+import TabContainer from '@ui5/webcomponents/dist/TabContainer.js';
+import { DetailedHTMLProps, HTMLAttributes, MouseEventHandler } from 'react';
+
+interface ObjectPageAnchorTabPropTypes extends HTMLAttributes<HTMLElement> {
+  text: string;
+  'with-sub-sections'?: boolean;
+  selected?: boolean;
+}
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace JSX {
+    interface IntrinsicElements {
+      'ui5-object-page-anchor-tab': DetailedHTMLProps<ObjectPageAnchorTabPropTypes, HTMLElement>;
+    }
+  }
+}
+
+const metadata = {
+  tag: 'ui5-object-page-anchor-tab',
+  properties: {
+    withSubSections: {
+      type: Boolean
+    }
+  },
+  events: {
+    'show-sub-sections': {}
+  }
+};
+
+class ObjectPageAnchorTab extends Tab {
+  static get metadata() {
+    return metadata;
+  }
+
+  static async onDefine() {
+    await Icon.define();
+  }
+
+  static get stripTemplate() {
+    return (context, tags, suffix) => {
+      setTags(tags);
+      setSuffix(suffix);
```

Comment:
Probably not a concern right now, but this code will not be affected by suffixing since internally used components (`ui5-icon`) are hard-coded.

The litHTML unsafe static function must be used to wrap all calls to `ui5-` tags in the template.